packages/grafana-ui/src/components/JSONFormatter/json_explorer/helpers.ts

```diff
@@ -2,10 +2,10 @@
 // License MIT, Copyright (c) 2015 Mohsen Azimi
 
 /*
- * Escapes `"` characters from string
+ * Escapes `"` characters from string for formatting
```

Comment:
**Nit:** I don't think this comment adds to much extra information

I would probably rather change the name of the function in this case to something like `escapeStringForFormatting()` (as it's more likely that a comment will remain unchanged when the logic of the function is extended than the function name itself), and also either get rid of the comment, or explain the _why_. 

I think what's unclear for anyone new to this piece of code is _why_ we need to do this.